docs/aws/README.md

```diff
@@ -1,11 +1,31 @@
-# AWS
+# Getting Started with Karpenter on AWS
 
-This guide will provide a complete Karpenter installation for AWS.
-These steps are opinionated and may need to be adapted for your use case.
+Karpenter automatically provisions new nodes in response to unscheduleable pods. Karpenter does this by observing events within the kubernetes cluster, and then sending commands to the underlying cloud platform. 
 
-> This guide should take less than 1 hour to complete and cost less than $.25
+In this example, the cluster is running on Amazon Web Services (AWS) Elastic Kubernetes Service (EKS). Karpenter is designed to be cloud platform independent, but currently only supports AWS. Contributions are welcomed. 
```

Comment:
consider "cloud platform independent" -> "vendor agnostic".

Reply:
can we do "cloud platform agnostic"? I'd rather be more specific than vendor.